specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2921 @@
+                }
+            ]
+        },
+        "HealthcareRelation": {
+            "type": "object",
+            "description": "Every relation is an entity graph of a certain relationType, where all entities are connected and have specific roles within the relation context.",
+            "required": [
+                "relationType",
+                "entities"
+            ],
+            "properties": {
+                "relationType": {
+                    "description": "Type of relation. Examples include: `DosageOfMedication` or 'FrequencyOfMedication', etc.",
+                    "type": "string",
+                    "enum": [
+                        "Abbreviation",
+                        "DirectionOfBodyStructure",
+                        "DirectionOfCondition",
+                        "DirectionOfExamination",
+                        "DirectionOfTreatment",
+                        "DosageOfMedication",
+                        "FormOfMedication",
+                        "FrequencyOfMedication",
+                        "FrequencyOfTreatment",
+                        "QualifierOfCondition",
+                        "RelationOfExamination",
+                        "RouteOfMedication",
+                        "TimeOfCondition",
+                        "TimeOfEvent",
+                        "TimeOfExamination",
+                        "TimeOfMedication",
+                        "TimeOfTreatment",
+                        "UnitOfCondition",
+                        "UnitOfExamination",
+                        "ValueOfCondition",
+                        "ValueOfExamination"
+                    ],
+                    "x-ms-enum": {
+                        "name": "relationType",
+                        "modelAsString": true
+                    }
+                },
+                "entities": {
+                    "description": "The entities in the relation.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/HealthcareRelationEntity"
+                    }
+                }
+            }
+        },
+        "HealthcareAssertion": {
+            "type": "object",
+            "properties": {
+                "conditionality": {
+                    "description": "Describes any conditionality on the entity.",
+                    "type": "string",
+                    "enum": [
+                        "hypothetical",
+                        "conditional"
+                    ],
+                    "x-ms-enum": {
+                        "name": "Conditionality",
+                        "modelAsString": false
+                    }
+                },
+                "certainty": {
+                    "description": "Describes the entities certainty and polarity.",
+                    "type": "string",
+                    "enum": [
+                        "positive",
```

Comment:
nit, do we have certain convention on the enum values? I can see camel-case, pascal-case, and upper-case separated by underscore in this single JSON.

Reply:
The convention we usually strive for (at least most recently) is Pascal case.  I think there may be some enums (such as for the `category` property in object `HealthcareEntityProperties`) which are represented with all-caps + underscore due to the fact that the values are returned directly from the underlying model.  In this specific example, the enum in the Swagger may not even be exhaustive due to the number of possible choices, which is likely why we also aren't transforming it from it's raw value from the model into something like pascal- or camel-case.

I can discuss with the team to see what the level of effort would be to unify on one standard, but not sure if we will have time to address it in this preview since it will require a lot of change on the service side.  I see Healthcare, Sentiment, Pii, and the async service all potentially being impacted.